console/console-init/ui/src/modules/address/dialogs/EditAddress/EditAddress.tsx

```diff
@@ -154,6 +160,28 @@ export const EditAddress: React.FunctionComponent = () => {
     }
   };
 
+  const getJsonPatchObject = (
+    op: string,
+    path: string,
+    value: string | null
+  ) => {
+    let patchObject: IPatchObject;
+    if (!value || value.trim() === "" || value.trim() === "null") {
```

Comment:
I think null check is wrong. It shoud be null `value.trim() ===null` rather than string